### PR TITLE
Upgrade to bootstrap5

### PR DESCRIPTION
**Pull request by Katrienvh**
## Type

- Enhancement

## Pull request description

Upgrade frontend/backend to Bootstrap 5 beta.

Most importend changes I did: 
- Input group does not need an extra wrapper link pre/append.
- Removed `.form-row` because its deprecated and used new gutter utility classes instead.
- `.form-group` is deprecated, but made our own custom class so we can still use is.
- All data attributes that trigger the Bootstrap javascipt are now prefixed with `data-bs-`
- Spacing utility classes are changed. `.ml` (Margin Left) and `.mr` (Margin Right) are now `.ms` (margin start = left) en `me` (margin end = right). This counts for al the classes that use the same structure.
- Removed `.form-inline` because it's deprecated and use flexbox utility classes instead.
- All custom form element stylings are now the default stylings, so removed the deprecated custom styling classes.
- Bootstrap now uses `visually-hidden` instead of `sr-only`

This should resolve: https://github.com/forkcms/forkcms/issues/3295

### CODE REVIEW

File: src/Common/Core/Form.php [hunk 1]
```diff
@@ -243,7 +243,7 @@ public function addDropdown(
     ): SpoonFormDropdown {
         $name = (string) $name;
         $multipleSelection = (bool) $multipleSelection;
-        $class = (string) ($class ?? 'form-control fork-form-select');
```
Comment (carakas) [inline src/Common/Core/Form.php:246]: Was there styling on this class that needs to be removed?

**Review by carakas — APPROVED** [review]
LGTM

File: src/Common/Core/Form.php (hunk 1 above)
Comment (Katrienvh) [reply src/Common/Core/Form.php:246]: Nope, `form-control` was from bootstrap and `fork-form-select` is an old class that was used long ago in fork

**Comment by Katrienvh** [comment]
To be fixed: installer -> some layout is a little broken

**Comment by carakas** [comment]
@Katrienvh there is a test looking for <label for="file" class="custom-file-label"> but failing because of the new layout. Can you update the test so that it looks for the correct HTML of the label?

**Review by carakas — CHANGES_REQUESTED** [review]
Failing test needs to be fixed first by updating the html

**Review by carakas — APPROVED** [review]

**Pull request merged**